schema/ClimateInformation.schema.yaml

```diff
@@ -438,17 +438,17 @@ ClimateTimeSeries:
       Type: "Group(TimeSeries)"
       Description: Snow depth
       Notes: Current observed depth of snow on the ground.
-    albedo:
+    albedo_ground:
       Constraints:
-      - display_name="Albedo"
+      - display_name="Albedo - Ground"
```

Comment:
Anything wrong with "Ground Albedo" (`ground_albedo`)? That reads more naturally.